controller/model/posture_check_manager.go

```diff
@@ -144,7 +145,10 @@ func (self *PostureCheckManager) Marshall(entity *PostureCheck) ([]byte, error)
 		RoleAttributes: entity.RoleAttributes,
 	}
 
-	entity.SubType.fillProtobuf(msg)
+	if entity.SubType != nil {
```

Comment:
Interesting, subtype should never be nil.

Reply:
Unless this is to account for a partial update? Say a name change only?

Reply:
Yes, I think it happened in one of the acceptance tests with a PATCH